src/pages/admin-ui-sdk/extension-points/banner-notification.md

```diff
@@ -0,0 +1,60 @@
+---
+title: bannerNotification
+description: Create a banner notification mass action in the Adobe Commerce Admin.
+keywords:
+  - App Builder
+  - Extensibility
+---
+
+# bannerNotification
+
+The `bannerNotification` extension point defines the contents of banner notifications that display when attempting a customer, order, or product mass action.
```

Comment:
bannerNotification works also for order view button. I updated the wiki with an example. 
https://wiki.corp.adobe.com/display/DMSArchitecture/bannerNotification